### PR TITLE
Uavsar tutorial sections

### PR DESCRIPTION
**Pull request by ZachHoppinen**
Split tutorial into sections

### CODE REVIEW

**Comment by scottyhq** [comment]
Is this replacing #57 ? Should we close that one and focus here?

**Comment by ZachHoppinen** [comment]
Yes, I am gonna close that one. I am trying to create a multi-part tutorial like the database and git ones so I copied the cells from the #57 to this branch.

File: book/tutorials/database/3_forming_queries.ipynb [hunk 1]
```diff
@@ -104,12 +104,7 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {
-    "tags": [
-     "nbsphinx-gallery",
-     "nbsphinx-thumbnail"
-    ]
-   },
+   "metadata": {},
```
Comment (scottyhq) [inline book/tutorials/database/3_forming_queries.ipynb]: not sure why the metadata changed in all the database notebooks

**Comment by scottyhq** [comment]
Great! I'd prefer not to add the `data` folder here since it's 50+MB. I made a copy of those files in `/home/jovyan/shared/uavsar-tutorial-data`, but you could also create a separate repository (like `snowex/uavsar-tutorial-data`) and add them there (then the first step in your tutorial could be `cd /tmp; git clone https://github.com/snowex/uavsar-tutorial-data.git`

**Comment by ZachHoppinen** [comment]
Hey @scottyhq  created that github repo and am cloning the data to tmp now.

File: book/tutorials/database/3_forming_queries.ipynb (hunk 1 above)
Comment (ZachHoppinen) [reply book/tutorials/database/3_forming_queries.ipynb]: I think when I was cleaning notebooks I ran the clean *.ipynb command in the wrong directory. I reverted all these files back to the commit before I did that.

**Comment by scottyhq** [comment]
to make the spellchecker happy you can add `hist` to the following list: https://github.com/snowex-hackweek/website2022/blob/85bf6a1449a0f42c4d27ac9c50ef3ab043f2da74/.github/workflows/qaqc.yaml#L36

**Comment by scottyhq** [comment]
this looks great! Happy to merge whenever @ZachKeskinen 

* `from uavsar_pytools.uavsar_tools import create_netrc` see https://github.com/snowex-hackweek/website2022/issues/60#issuecomment-1168790387 for an alternative,  

adding `%config InlineBackend.figure_format='retina'` to your imports generally makes the tick labels and titles look better on most modern screens

**Comment by ZachHoppinen** [comment]
Hey @scottyhq can we pull this in?

**Review by scottyhq — APPROVED** [review]
thanks @ZachKeskinen !

**Pull request merged**